mitxgraders/listgrader.py

```diff
@@ -633,37 +640,29 @@ def schema_config(self):
         """Define the configuration options for SingleListGrader"""
         # Construct the default ItemGrader schema
         schema = super(SingleListGrader, self).schema_config
-        # Append options, replacing the ItemGrader 'answers' key
+        # Append options
         return schema.extend({
             Required('ordered', default=False): bool,
             Required('length_error', default=False): bool,
             Required('missing_error', default=True): bool,
             Required('delimiter', default=','): text_string,
             Required('partial_credit', default=True): bool,
-            Required('subgrader'): ItemGrader,
-            Required('answers', default=[]): Any(list, (list,))  # Allow for a tuple of lists
+            Required('subgrader'): ItemGrader
         })
+        # Note that we use the ItemGrader definitions for 'answers', although
+        # validate_expect is shadowed, and we do post-processing in post_schema_validation
 
     # Make sure that the ItemGrader validate_expect isn't used
     validate_expect = None
```

Comment:
This `validate_expect = None` should be deleted now, right?

Reply:
Absolutely. Didn't even realize that was there.